src/sonic-config-engine/tests/simple-sample-graph-metadata.xml

```diff
@@ -253,6 +253,11 @@
      <a:Reference i:nil="true"/>
      <a:Value>10.0.10.7;10.0.10.8</a:Value>
     </a:DeviceProperty>
+    <a:DeviceProperty>
+     <a:Name>SubRole</a:Name>
+     <a:Reference i:nil="true"/>
+     <a:Value>sub_role_1</a:Value>
```

Comment:
Can you please elaborate what this Property signifies ?

Reply:
We will receive specific SubRole DeviceProperty values and use the same for policy enforcement.

Reply:
So now we have two subroles. One of them was introduced by multiasic efforts and another one is part of the minigraph.
I'd suggest to rename multinpu subrole to something else, otherwise we will always confuse both of them.

Reply:
Since the new property for the whole device, can rename this to device_role. The sub_role can remain the roles of each asic in a device

Reply:
we already have "type" (Leaf/Tor/Spine)  as part of DEVICE_METADATA. May be have sub_type as part of DEVICE_METADATA

Reply:
we cannot have two sub_role

Reply:
Just to be most precise we are looking at what to name the final DEVICE_METADATA value? We are not looking at changing SubRole to some other element in the minigraph. 

I see 2 suggestions here, one is changing the sub_role for multi_asic, this could be multi_asic_sub_role to be most explicit. The other suggestion is changing sub_role for minigraph to something like what @arlakshm suggested: device_subrole/device_role.